CONTRIBUTING.md

```diff
@@ -6,17 +6,34 @@ The [list of issues](https://github.com/mimblewimble/grin/issues) is a good plac
 
 Additional tests are rewarded with an immense amount of positive karma.
 
-More documentation or updates/fixes to existing documentation are also very welcome. However, if submitting a PR consisting of documentation changes only, please try to ensure that the change is significantly more substantial than one or two lines. For example, working through an install document and making changes and updates throughout as you find issues is worth a PR. For typos and other small changes, either contact one of the developers, or if you think it's a significant enough error to cause problems for other users, please feel free to open an issue.
+More documentation or updates/fixes to existing documentation are also very welcome. However, if submitting a PR(Github Pull-Request) consisting of documentation changes only, please try to ensure that the change is significantly more substantial than one or two lines. For example, working through an install document and making changes and updates throughout as you find issues is worth a PR. For typos and other small changes, either contact one of the developers, or if you think it's a significant enough error to cause problems for other users, please feel free to open an issue.
 
-Find us:
+# Find Us
 
+If any help is needed during your effort to contribute on this project, please don't hesitate to contact us:
 * Chat: [Gitter](https://gitter.im/grin_community/Lobby).
+* [Forum](https://www.grin-forum.org/)
+* [Website](https://grin-tech.org)
 * Mailing list: join the [~MimbleWimble team](https://launchpad.net/~mimblewimble) and subscribe on Launchpad.
+* News: [Twitter](https://twitter.com/grinmw). Twitter bot that scrapes headlines, mailing list, and reddit posts related to MimbleWinble/Grin.
+
+# Pull-Request Name Prefix
+
+Please consider putting one of the following prefixes in the title of your pull-request:
+- **feat**:     A new feature
+- **fix**:      A bug fix
+- **docs**:     Documentation only changes
+- **style**:    Formatting, missing semi-colons, white-space, etc
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+- **perf**:     A code change that improves performance
+- **test**:     Adding missing tests
+- **chore**:    Maintain. Changes to the build process or auxiliary tools/libraries/documentation
+
+For example: `fix: a panick on xxx when grin exiting`. Please don't worry if you can't find a suitable prefix, this's just optional, not mandatory.
 
 # Grin Style Guide
 
-Grin uses `rustfmt` to maintain consistent formatting.
-Please use the git commit hook as explained below.
+Grin uses `rustfmt` to maintain consistent formatting, and we're using the git commit hook as explained below.
```

Comment:
Same here not sure this changes is needed or belong in this PR.

Reply:
😸